### PR TITLE
Update signal mutation model to fit new data

### PR DESCRIPTION
**Pull request by leexgh**
Related to https://github.com/knowledgesystems/signal/issues/69

### CODE REVIEW

**Review by onursumer — CHANGES_REQUESTED** [review]
Thanks! A few requests regarding the field names and mixins.

File: web/src/main/java/org/cbioportal/genome_nexus/web/mixin/SignalMutationMixin.java [hunk 1]
```diff
@@ -44,4 +46,13 @@
 
     @ApiModelProperty("QC Pass Counts by Tumor Type")
     private List<CountByTumorType> qcPassCountsByTumorType;
+
+    @ApiModelProperty("general_population_stats")
```
[the remark this answers is not in the document]
Comment (leexgh) [reply web/src/main/java/org/cbioportal/genome_nexus/web/mixin/SignalMutationMixin.java]: @onursumer Good point. For some fields I couldn't come up with a human-readable description, could you give some suggestions and also double check those I added?

[the remark this answers is not in the document]
Comment (onursumer) [reply web/src/main/java/org/cbioportal/genome_nexus/web/mixin/SignalMutationMixin.java]: Thanks! I think they look good in general.

Which fields are the ones you couldn't come up with a description? Certain field names are not very clear to me either, so most likely I couldn't come up with anything better. Maybe @inodb can help to improve those descriptions? It's not a big deal at this point though. We can always update the descriptions later.

File: web/src/main/java/org/cbioportal/genome_nexus/web/mixin/GeneralPopulationStatsMixin.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.cbioportal.genome_nexus.web.mixin;
+
+import org.cbioportal.genome_nexus.model.SignalPopulationStats;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class GeneralPopulationStatsMixin {
+    @ApiModelProperty("Counts")
```
Comment (leexgh) [inline web/src/main/java/org/cbioportal/genome_nexus/web/mixin/GeneralPopulationStatsMixin.java:8]: Is "Counts" enough?

File: web/src/main/java/org/cbioportal/genome_nexus/web/mixin/GeneralPopulationStatsMixin.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    private SignalPopulationStats counts;
+
+    @ApiModelProperty("Frequencies")
```
Comment (leexgh) [inline web/src/main/java/org/cbioportal/genome_nexus/web/mixin/GeneralPopulationStatsMixin.java:11]: Is "Frequencies" enough?

File: web/src/main/java/org/cbioportal/genome_nexus/web/mixin/HrdScoreMixin.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package org.cbioportal.genome_nexus.web.mixin;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class HrdScoreMixin {
+    @ApiModelProperty("Median HRD LST")
```
Comment (leexgh) [inline web/src/main/java/org/cbioportal/genome_nexus/web/mixin/HrdScoreMixin.java:6]: Not sure if we should add "Median" here, same for others below

File: web/src/main/java/org/cbioportal/genome_nexus/web/mixin/SignalPopulationStatsMixin.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.genome_nexus.web.mixin;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class SignalPopulationStatsMixin {
+    @ApiModelProperty("Impact")
+    private Double impact;
+
+    @ApiModelProperty("European")
```
Comment (leexgh) [inline web/src/main/java/org/cbioportal/genome_nexus/web/mixin/SignalPopulationStatsMixin.java:9]: The name is referred from here: https://gnomad.broadinstitute.org/faq#how-are-population-names-abbreviated
Does SIGNAL have the same pattern?

File: web/src/main/java/org/cbioportal/genome_nexus/web/mixin/StatsByTumorTypeMixin.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    @ApiModelProperty("Number Of Cancer Type Count")
+    private Integer nCancerTypeCount;
+
+    @ApiModelProperty("Frequency Of Cancer Type Count")
+    private Double fCancerTypeCount;
+
+    @ApiModelProperty("Frequency Of Biallelic")
+    private Double fBiallelic;
+
+    @ApiModelProperty("Median Age at Dx")
```
Comment (leexgh) [inline web/src/main/java/org/cbioportal/genome_nexus/web/mixin/StatsByTumorTypeMixin.java:19]: Or "Median Age at diagnosis"?

**Review by leexgh — COMMENTED** [review]
@inodb Could you take a look at the following description and let me know if there is a better one for them? It would be good if you can double check others also

**Review by inodb — APPROVED** [review]

**Comment by inodb** [comment]
We can address the descriptions later, thanks @leexgh !

**Review by onursumer — APPROVED** [review]

**Pull request merged**